shim/src/sanitize/accessors.js

```diff
@@ -0,0 +1,62 @@
+// Adapted from SES/Caja - Copyright (C) 2011 Google Inc.
+// https://github.com/google/caja/blob/master/src/com/google/caja/ses/startSES.js
+// https://github.com/google/caja/blob/master/src/com/google/caja/ses/repairES5.js
+
+import { getPrototypeOf, defineProperty, getOwnPropertyDescriptor } from '../commons';
+
+// Fix legacy accessors to comply with strict mode and ES2016 semantics,
+// we need to redefine them while in strict mode.
+// https://tc39.github.io/ecma262/#sec-object.prototype.__defineGetter__
+
+export default function sanitizeAccessors(sandbox) {
+
+  const { confinedWindow: { Object: { prototype: proto } } } = sandbox;
```

Comment:
if the role of this function is to sanitize accessors, then pass the `Object.prototype` to it instead of passing sandbox, which is our own invention.